cni/network/plugin/main.go

```diff
@@ -182,9 +182,12 @@ func rootExecute() error {
 		cniReport.GetReport(pluginName, version, ipamQueryURL)
 
 		var upTime time.Time
-		upTime, err = platform.GetLastRebootTime()
+		p := platform.NewExecClient(logger)
+		upTime, err = p.GetLastRebootTime()
 		if err == nil {
 			cniReport.VMUptime = upTime.Format("2006-01-02 15:04:05")
+		} else {
+			logger.Error("Failed to query last reboot time", zap.Error(err))
```

Comment:
This should be logged in platform package right ?

Reply:
solved